matrix/matrix_class.py

```diff
@@ -351,6 +373,46 @@ def __pow__(self, other):
     def dot_product(cls, row, column):
         return sum(row[i] * column[i] for i in range(len(row)))
 
+    def cross_product(self, other: "Matrix") -> "Matrix":
+
+        # https://en.wikipedia.org/wiki/Cross_product
+        if isinstance(other, Matrix):
+            if other.order == self.order == (2, 1):
+                return Matrix(
+                    [
+                        [0],
+                        [0],
+                        [
+                            (self.rows[0][0] * other.rows[1][0])
+                            - (self.rows[1][0] * other.rows[0][0])
+                        ],
+                    ]
+                )
+            elif other.order != (3, 1) or self.order != (3, 1):

```

Comment:
Why are we limiting the function to perform cross-product on two orders? This limits the functionality of the class.